leaderboard/contributors/migrations/0012_auto_20160428_1827.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.contrib.gis.db.models.fields
+
+
+def remove_empty_points(apps, schema_editor):
+    Contribution = apps.get_model('contributors', 'Contribution')
+    db_alias = schema_editor.connection.alias
+    Contribution.objects.using(db_alias).filter(point=None).delete()
+
+def noop(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contributors', '0011_auto_20160208_2034'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='contribution',
+            name='country',
+        ),
+        migrations.AddField(
+            model_name='contribution',
+            name='point',
+            field=django.contrib.gis.db.models.fields.PointField(srid=4326, null=True, blank=True),
```

Comment:
I realize that the Point field is new, so it needs to be nullable - but should we autofill in some precomputed values based on the country and fill blank spots in the dataset?